java/client/test/org/openqa/selenium/remote/tracing/opentelemetry/TracerTest.java

```diff
@@ -118,11 +131,13 @@ public void shouldBeAbleToCreateASpanWithEvents() {
         .isEqualTo("Test event start");
     assertThat(timedEvents).element(0).extracting(SpanData.Event::getTotalAttributeCount).isEqualTo(0);
     assertThat(timedEvents).element(1).extracting(SpanData.Event::getName).isEqualTo("Test event end");
-    assertThat(timedEvents).element(1).extracting(SpanData.Event::getTotalAttributeCount).isEqualTo(6);
+    assertThat(timedEvents).element(1).extracting(SpanData.Event::getTotalAttributeCount).isEqualTo(10);
```

Comment:
This is why reusing test cases is a less than ideal idea. Prefer lots of small tests than mashing things into a single test case. That allows test names to be descriptive. In the ideal world, we'd be able to delete the class under test and all the test methods, leaving just the test method signatures, and create something functionally equivalent.